make

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#Operate in current direcory
+cd $(dirname $0)
+
+# Variables
+LANG="python3"
+MINIFY=0
+DEBUG=0
+
+#Ensure npm installed
+NPM="$(which npm)"
+if [ "$NPM" == "" ]; then
+	echo "npm not installed"
```

Comment:
I'm a tad confused here. Are you saying the script will not run if you use, say, 4-space indents?